lms/static/js/learner_dashboard/spec/program_details_view_spec.js

```diff
@@ -473,11 +468,23 @@ describe('Program Details Header View', () => {
                 },
             ],
         },
+        subscriptionData: [
+            {
+                trial_end: '1970-01-01T03:25:45Z',
+                next_payment_date: '1970-06-03T07:12:04Z',
+                price: '100.00',
+                currency: 'USD',
+                subscription_state: 'pre',
```

Comment:
I know this is a test file, just curious, is `'pre'` one of the states we'll have?

Reply:
Not really, any state that is not 'active' or 'inactive' will just show the ui of the initial state (i.e the one with start-7day free trial) so this test verifies that